site/src/util/mui-utils.tsx

```diff
@@ -16,7 +16,7 @@ export const parseIntFromPixelString = (pixelString: string): number => {
  * components modal, drawer, menu, popover, dialog.
  */
 export const useScrollLock = (active: boolean) =>
-  useLayoutEffect(() => {
+  useEffect(() => {
```

Comment:
Prevents this warning:

```
Warning: useLayoutEffect does nothing on the server, because its effect cannot be encoded into the server renderer's output format. This will lead to a mismatch between the initial, non-hydrated UI and the intended UI. To avoid this, useLayoutEffect should only be used in components that render exclusively on the client. See https://reactjs.org/link/uselayouteffect-ssr for common fixes.
```

If `useEffect` is not what we need here, we can switch to adding `next/dynamic` around `Modal` and `Popover`. I haven't spotted any visual regressions with the current approach though.